bootstrap

```diff
@@ -1,159 +1,91 @@
-    exit 1
-fi
-
-# Use the newest version we find
-if [ -z "$PYTHON" ]; then
-  for i in 12 11 10; do
-    command -v "python3.$i" && PYTHON="python3.$i" &>/dev/null && break
-  done
-fi
-if [ -z "$PYTHON" ]; then
-  # This would be bizarre, but I suppose possible
-  PYTHON=${PYTHON:-"python3"}
+    if [ "$1" = "install" ]; then
+        install=true
+    else
+        echo "Invalid command, supported commands are: 'install'"
+        exit 1
+    fi
 fi
-echo "Using python: $PYTHON"
 
 case "$(uname -s)" in
 Linux)
-    if command -v lsb_release; then
+    if command -v lsb_release > /dev/null; then
         OS=$(lsb_release --id --short)
     else
         . /etc/os-release
-        OS=$(echo $NAME | tr -d ' ')
+        OS=$(echo "$NAME" | tr -d ' ')
     fi
-    # rpm/dnf is the default, to reduce repetition in the case statement
-    has_pkg="rpm -q --whatprovides"
-    install_pkg="sudo dnf install -y"
     case "$OS" in
     Ubuntu|Debian|LinuxMint)
-        deps=(qemu-utils python3-dev libssl-dev python3-pip python3-wheel python3-venv libev-dev libvirt-dev libffi-dev libyaml-dev build-essential jq curl)
-        has_pkg="dpkg -s"
+        deps=(qemu-utils python3-dev libssl-dev libev-dev libvirt-dev libffi-dev libyaml-dev pipx build-essential jq curl)
+        has_pkg="dpkg -C"
         install_pkg="sudo apt install -y"
         ;;
-    RedHatEnterpriseWorkstation|RedHatEnterpriseServer|RedHatEnterprise|CentOS)
-        deps=(python39-pip python39-devel mariadb-devel libev-devel libvirt-devel libffi-devel)
-        ;;
-    CentOSStream)
-        PYTHON=python3.12
-        deps=($PYTHON-pip $PYTHON-devel)
-        ;;
-    AlmaLinux|RockyLinux)
-        PYTHON=python3.12
-        deps=($PYTHON-pip $PYTHON-devel libev-devel libvirt-devel libffi-devel)
-        ;;
-    Fedora|FedoraLinux)
-        deps=($PYTHON-pip $PYTHON-devel libev-devel libvirt-devel libffi-devel)
+    RedHatEnterpriseWorkstation|RedHatEnterpriseServer|RedHatEnterprise|CentOS|CentOSStream|AlmaLinux|RockyLinux|Fedora|FedoraLinux)
+        # Use the newest version we find
+        if [ -z "$PYTHON" ]; then
+            for i in 13 12 11 10; do
+                command -v "python3.$i" > /dev/null && PYTHON="python3.$i" && break
+            done
+        fi
+        if [ -z "$PYTHON" ]; then
+            # This would be bizarre, but I suppose possible
+            PYTHON=${PYTHON:-"python3"}
+        fi
+        # rpm/dnf is the default, to reduce repetition in the case statement
+        has_pkg="rpm -q --whatprovides"
+        install_pkg="sudo dnf install -y"
+        deps=(libev-devel libffi-devel libvirt-devel "$PYTHON-devel" pipx)
         ;;
     "openSUSE project"|"SUSE LINUX"|"openSUSE"|"openSUSELeap"|"openSUSETumbleweed")
-        PYTHON=python3.12
-        deps=(python312-pip python312-devel python312 libev-devel libvirt-devel libffi-devel)
+        deps=(python python-devel libev-devel libffi-devel libvirt-devel python-pipx)
+        has_pkg="rpm --whatprovides"
```

Comment:
Missing `-q`. However, SUSE users mostly not using `rpm` directly, there are zypper commands for that purpose.